package.json

```diff
@@ -29,7 +29,7 @@
   "devDependencies": {
     "@babel/core": "^7.15.0",
     "@babel/preset-env": "^7.15.0",
-    "@vercel/ncc": "^0.29.0",
+    "@vercel/ncc": "^0.33.3",
```

Comment:
whats the reasoning behind bumping ncc?

Reply:
no reasoning, but just thinking to use the latest.

Reply:
maybe this one in mind, https://github.com/vercel/ncc/pull/801

Reply:
sounds good!